src/ui/components/Card/index.js

```diff
@@ -12,10 +12,25 @@ export default class Card extends React.Component {
     footerLink: PropTypes.node,
     footerText: PropTypes.node,
     header: PropTypes.node,
+    photonStyle: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    // New-style cards that we'll roll out gradually so we don't break
```

Comment:
I'm not sure exactly what this is saying. I think it would help if it were rewritten as a complete sentence.